backend/src/main/java/com/woowacourse/gongcheck/core/presentation/HostController.java

```diff
@@ -37,4 +38,12 @@ public ResponseEntity<EntranceCodeResponse> showEntranceCode(@AuthenticationPrin
         String entranceCode = hostService.createEntranceCode(hostId);
         return ResponseEntity.ok(EntranceCodeResponse.from(entranceCode));
     }
+
+    @GetMapping("/hosts/me")
+    @HostOnly
+    public ResponseEntity<HostProfileResponse> showProfile(@AuthenticationPrincipal final Long hostId) {
+        final HostProfileResponse response = hostService.findProfile(hostId);
```

Comment:
```suggestion
        HostProfileResponse response = hostService.findProfile(hostId);
```
메소드 변수에 final을 달지 않는게 컨벤션이었습니다 :)